autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -54,7 +54,9 @@ impl UnidentifiedClient {
     ) -> Result<(WebPushClient, impl IntoIterator<Item = ServerMessage>), SMError> {
         trace!("❓UnidentifiedClient::on_client_msg");
         let ClientMessage::Hello {
-            uaid, broadcasts, ..
+            uaid,
+            broadcasts,
+            channel_ids,
```

Comment:
We ignore this because it's essentially deprecated: Desktop no longer passes in `channel_ids` here, so there's no point in metricing it.

Reply:
Argh. right. Forgot about that. Fortunately, looks like we can take advantage of the tty PR to do this reasonably efficiently.